Makefile.am

```diff
@@ -46,49 +46,23 @@ LDADD = \
     $(TCTI_DEV_LIBS) $(CRYPTO_LIBS)
 
 bin_PROGRAMS = \
-    tools/tpm2_create \
-    tools/tpm2_createprimary \
-    tools/tpm2_load \
-    tools/tpm2_send_command \
-    tools/tpm2_dump_capability \
-    tools/tpm2_pcrlist \
-    tools/tpm2_listpersistent \
-    tools/tpm2_startup \
-    tools/tpm2_rc_decode \
-    tools/tpm2_verifysignature \
-    tools/tpm2_getmanufec \
-    tools/tpm2_quote \
-    tools/tpm2_takeownership \
-    tools/tpm2_getpubek \
-    tools/tpm2_getpubak \
-    tools/tpm2_akparse \
-    tools/tpm2_hash \
-    tools/tpm2_activatecredential \
-    tools/tpm2_makecredential \
-    tools/tpm2_nvlist \
-    tools/tpm2_nvread \
-    tools/tpm2_nvreadlock \
-    tools/tpm2_nvwrite \
-    tools/tpm2_nvdefine \
-    tools/tpm2_nvrelease \
-    tools/tpm2_hmac \
-    tools/tpm2_certify \
-    tools/tpm2_readpublic \
-    tools/tpm2_getrandom \
-    tools/tpm2_encryptdecrypt \
-    tools/tpm2_evictcontrol \
-    tools/tpm2_loadexternal \
-    tools/tpm2_rsadecrypt \
-    tools/tpm2_rsaencrypt \
-    tools/tpm2_sign \
-    tools/tpm2_unseal \
-    tools/tpm2_dictionarylockout \
-    tools/tpm2_createpolicy \
-    tools/tpm2_pcrextend \
-    tools/tpm2_pcrevent
+    tools/tpm2_hash
+
```

Comment:
This will be a huge commit since all the tools will have to be converted in the same patch to maintain git bisect ability.

I wonder if the changes could be split in some way to prevent this, but probably not.

Reply:
Yeah this is going to be a big ugly patch as far as I can tell.